### DISCUSSION TITLE
How do i work with socks4/socks5 proxies?

### DISCUSSION THREAD

**Discussion by FZKiritsugu** [aio-libs/aiohttp#5126] [Q&A] [answered]
^Title
hello,
I heard that aiohttp is not supporting socks proxies.
i want to import proxies from a list (socks4/socks5) and each request to use a random proxy.

**Comment by SomberNight** [comment, marked as answer]
You can use [aiohttp-socks](https://github.com/romis2012/aiohttp-socks).
